### PR TITLE
IDL docs.

### PR DESCRIPTION
**Pull request by bkaradzic**

### CODE REVIEW

File: bgfx.idl [hunk 1]
```diff
@@ -1164,87 +1164,89 @@ func.calcTextureSize
...
-	                              --- - `BGFX_SAMPLER_[U/V/W]_[MIRROR/CLAMP]` - Mirror or clamp to edge wrap
-	                              ---   mode.
-	                              --- - `BGFX_SAMPLER_[MIN/MAG/MIP]_[POINT/ANISOTROPIC]` - Point or anisotropic
-	                              ---   sampling.
-	.skip  "uint8_t"              --- Skip top level mips when parsing texture.
-	.info  "TextureInfo*" { out } --- When non-`NULL` is specified it returns parsed texture information.
+	"TextureHandle"                            --- Texture handle.
+	.mem   "const Memory*"                     --- DDS, KTX or PVR texture binary data.
+	.flags "uint64_t"                          --- Texture creation (see `BGFX_TEXTURE_*`.), and sampler (see `BGFX_SAMPLER_*`)
+	 { "BGFX_TEXTURE_NONE|BGFX_SAMPLER_NONE" } --- flags. Default texture sampling mode is linear, and wrap mode is repeat.
```
Comment (cloudwu) [inline bgfx.idl:1170]: It should be `{ default = "BGFX_TEXTURE_NONE|BGFX_SAMPLER_NONE" }`

File: bgfx.idl [hunk 2]
```diff
@@ -1546,8 +1558,11 @@ func.setViewClear
 	.flags   "uint16_t" --- Clear flags. Use `BGFX_CLEAR_NONE` to remove any clear
 	                    --- operation. See: `BGFX_CLEAR_*`.
 	.rgba    "uint32_t" --- Color clear value.
+	 { default = 0x000000ff }
```
Comment (cloudwu) [inline bgfx.idl]: You may want to use `"0x000000ff"` here,  because `0x000000ff` will convert to `255` by generator.

Comment (bkaradzic) [reply bgfx.idl]: Changed.

File: bgfx.idl (hunk 1 above)
Comment (bkaradzic) [reply bgfx.idl:1170]: Fixed.

**Pull request merged**